contents/tutorials/churn-rate.md

```diff
@@ -0,0 +1,87 @@
+
+## Visualizing churn rate
+
+To start visualizing churn rate, you can set up a retention funnel for users completing an action or event, then see how many continue their usage. The action or event you choose could be something like an autocapture of a pageview, an event capture you’ve written into your code or a combination of events (an action) that define feature usage. For us, we’ll set up a retention chart for unique users who completed the event `downloaded_file` over the last 8 weeks. 
+
+![Retention chart](../images/tutorials/churn-rate/retention-chart.png)
+
+We can calculate our weekly churn and total churn by comparing our initial cohort size with the percentage of users remaining. For example, if we care about weekly churn, we can compare Week 0 and Week 1. If we care about churn across the entire period, we can compare Week 0 and Week 8. To learn more about retention charts, check out our [product manual](https://posthog.com/manual/retention).
+
+### Lifecycle charts
+
+Another way to visualize churn is through lifecycle charts. Lifecycle charts show us, in a time period, how many users:
+
+- do an event for the first time
+- returned to do that event
+- didn’t do the event
+- resurrect to do the event after not doing it in the previous period
+
+The dormant users are the ones who churned. Comparing them with the ones who did the event (new, returning, or resurrected) can give you the churn rate for that event or action.
+
+![Lifecycle chart](../images/tutorials/churn-rate/lifecycle-chart.png)
+
+Once set up, we can click on the dormant part of the chart to get a list of dormant (churned) users. We can see user properties and, more importantly, session recordings for these users. This helps us lower churn in the next section.
+
+You can learn more about lifecycle charts in our [product manual](https://posthog.com/manual/lifecycle). 
+
+## Lowering churn
+
+Now that we calculated and visualized the churn rate in our product, we can figure out ways to lower it. Being an all-in-one product OS, PostHog has tools to help you with this, and they connect to your data and the work you’ve already done.
+
+### Session recordings
+
+First on this list is session recordings. From any of the visualizations, we can get a list of users who churned at specific times. Clicking on the charts we created earlier in this tutorial gives us a list of users. We can check this list for recordings of recent sessions of churned users.
+
+Once we find a session recording from a churned user, we can watch their actions leading up to them churning. We might discover, for example, that the pricing page confuses users or that they can’t set up a key feature. We can use this information to improve these problem areas (check out [experiments](https://posthog.com/manual/experimentation)).
+
+### Cohorts
```

Comment:
There's actually an easier way to do this, though we should explain both. You can create a cohort directly from the list of persons generated when you click on the result of an insight. 

![Screenshot 2022-09-29 at 12 29 02](https://user-images.githubusercontent.com/92976667/193020128-19eeeca4-1dcb-497e-82a0-3d833604f151.png)

Reply:
Ahh yeah, added this.